PWGHF/D2H/Tasks/taskDstarToD0Pi.cxx

```diff
@@ -179,8 +186,11 @@ struct HfTaskDstarToD0Pi {
       auto collision = candDstar.collision_as<CollisionsWCent>();
       auto centrality = collision.centFT0M(); // 0-100%
 
+      registry.fill(HIST("Efficiency/hPVContributorsEventsWCand"), collision.numContrib(), centrality);
```

Comment:
This should be filled per collision, not per candidate (i.e. it has to be moved outside the candidate loop)
Also, remember to rename it as suggested above